src/constants/index.ts

```diff
@@ -1,3 +1,45 @@
+/**
+ * Months Enum.
+ *
+ * @name Month
+ * @constant
+ * @type {Object.<string, number>}
+ * @default
+ */
+
+export const Month = {
+  Jan : 0,
+  Feb : 1,
+  Mar : 2,
+  Apr : 3,
+  May : 4,
+  Jun : 5,
+  Jul : 6,
+  Aug : 7,
+  Sep : 8,
+  Oct : 9,
+  Nov : 10,
+  Dec : 11,
+}
```

Comment:
Suggest adding a const assertion here and for DayofWeek

```suggestion
} as const
```

https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-4.html#const-assertions